src/docs/release/breaking-changes/use-maxLengthEnforcement-instead-of-maxLengthEnforced.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Use maxLengthEnforcement instead of maxLengthEnforced
+description: Introduing MaxLengthEnforcement enum.
+---
+
+## Summary
+
+_If you didn't use `maxLengthEnforced` in `TextField`/`TextFormField`/`CupertinoTextField`, or you don't care about the truncate behavior for the composing text editing value, you can stop reading._
```

Comment:
I think it is important to mention that the default value of `maxLengthEnforced` is true so a lot of people uses it implicitly by just setting the `maxLength` property. 

May be it is worth mentioning the `maxLenght` parameter as well (or even mention it in place of `maxLengthEnforced` since it is implicitly used by default).

Cc @justinmc @LongCatIsLooong @chunhtai (you might know better about what I'm trying to say).

Reply:
Yeah I think it's worth at least saying "the `maxLength` or `maxLengthEnforced` properties".

Users of `maxLength` will not be broken if they don't also use `maxLengthEnforced`, but it's good to know anyways.  They may want to change to using `MaxLengthEnforcement.allowComposingTextToFinish`.